posnums.c

```diff
@@ -0,0 +1,44 @@
+/* Can Parlar - Scans 2 numbers prints out their average wheather they are  prime  or not and wheather they are relativaly prime or not */
```

Comment:
Where does it print out whether it's relatively prime? also, when only one of the numbers if prime, it doesn't print out that it's prime.